src/scripts/main.js

```diff
@@ -358,3 +358,55 @@
 console.log(people); // you can remove it
 
 // write your code here
+
+const names = people.map((person) => person.name);
+
+const gender = people.map((person) => ({
+  ...person,
+  sex: person.sex === 'm' ? 'Male' : 'Female',
+}));
+
+const born = people.map((person) => person.born);
+const died = people.map((person) => person.died);
+
+const century = people.map((person) => Math.ceil(person.died / 100));
+
+const ages = people.map((person) => (person.died -= person.born));
+
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with class 'dashboard' exists in the DOM before attempting to append rows to it. If it doesn't exist, this line will cause an error.